etna/transforms/timestamp/fourier.py

```diff
@@ -60,6 +66,13 @@ def __init__(
             * if don't set, name will be ``transform.__repr__()``,
               repr will be made for transform that creates exactly this column
 
+        in_column:
+            name of column to work with:
+
+            * if ``in_column`` is ``None`` (default) both datetime and integer timestamps are supported;
+
+            * if ``in_column`` isn't ``None`` only numeric columns are supported
```

Comment:
This was done to performance reasons, using `determine_num_steps` here was very slow even on testing datasets.